server/sonar-search/src/main/java/org/sonar/search/EsSettings.java

```diff
@@ -81,7 +80,7 @@ Settings build() {
     return builder.build();
   }
 
-  private void configureFileSystem(ImmutableSettings.Builder builder) {
+  private void configureFileSystem(Settings.Builder builder) {
     File homeDir = props.nonNullValueAsFile(ProcessProperties.PATH_HOME);
     File dataDir;
     File workDir;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "workDir" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1481)